rfcs/text/0001-wildcard-data-type.md

```diff
@@ -115,56 +216,63 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
-ECS-compliant data consumers should not be impacted due to the `keyword` field family[2], however ECS-compliant producers will be required for their field mappings to use `wildcard` where defined by the spec. The ECS project will need to evaluate how best to
-evolve its tooling to support `wildcard`; wildcard will be the first time a Elastic licensed feature is specified in ECS.
-
 ### Ingestion
 
-Any component producing data (Beats, Logstash, third-party developed, etc.) will need to comply with the new mappings.
+Any component producing data (Beats, Logstash, third-party developed, etc.) will need to adopt the mappings in their index templates. The discussion around the handling of OSS vs. Basic field types between OSS and Basic modules/plugins will be handled outside of this proposal.
 
 ### Usage
```

Comment:
We already have another heading named "Usage", this threw me for a bit. Can we rename this heading?

Reply:
Absolutely 👍